qase-playwright/src/playwright.ts

```diff
@@ -182,6 +183,22 @@ qase.ignore = function() {
   return this;
 };
 
+/**
+ * Set a suite for the test case
+ * @param {string} value
+ * @example
+ * test('test', async ({ page }) => {
+ *    qase.suite("Suite");
+ *    await page.goto('https://example.com');
+ * });
+ */
+qase.suite = function(value: string) {
```

Comment:
Is it a suite in the root suite or on the same level as the root suite?

Reply:
It is test level suite. Root suite contains test suites.
Example:
'My root suite' -> 'Feature suite'